birdhouse/pavics-compose.sh

```diff
@@ -34,31 +30,38 @@ OPTIONAL_VARS='
   $EXTRA_PYWPS_CONFIG
   $SERVER_NAME
   $SERVER_DESCRIPTION
+  $SERVER_INSTITUTION
+  $SERVER_SUBJECT
+  $SERVER_TAGS
+  $SERVER_DOCUMENTATION_URL
+  $SERVER_RELEASE_NOTES_URL
+  $SERVER_SUPPORT_URL
+  $SERVER_LICENSE_URL
 '
 
 # we switch to the real directory of the script, so it still works when used from $PATH
 # tip: ln -s /path/to/pavics-compose.sh ~/bin/
 # Setup PWD for sourcing env.local.
-cd $(dirname $(readlink -f $0 || realpath $0))
+cd "$(dirname "$(readlink -f "$0" || realpath "$0")")" || (echo "Unresolved path [$0]" && exit 1)
```

Comment:
Also, if the `cd` fails then it will write a warning to stderr with basically the same information as the `echo`.

So I'd recommend either doing:

```sh
cd "$(dirname "$(readlink -f "$0" || realpath "$0")")" 2> /dev/null || echo "Unresolved path [$0]" && exit 1
```

to use your custom error message

or 

```sh
cd "$(dirname "$(readlink -f "$0" || realpath "$0")")" || exit 1
```

to use `cd`'s error message

Reply:
`cd "$(dirname "$(readlink -f "$0" || realpath "$0")")" || exit 1` seems fine to me.
As long as the error stops the execution instead of moving on silently.